testsuite/tests/grafana/test_dashboards.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for Grafana Dashboards definitions"""
+import pytest
+
+from testsuite.capabilities import Capability
+
+pytestmark = [
+        pytest.mark.skipif("TESTED_VERSION < Version('2.9')"),
+        pytest.mark.required_capabilities(Capability.OCP4),
+    ]
+
+# pylint: disable=line-too-long
+DASHBOARDS = [
+    ("apicast-mainapp",
+     "sum(rate(nginx_http_connections{namespace='$namespace', pod=~'apicast-$env.*'}[1m])) by (state)"),
+    ("apicast-services",
+     "sum(rate(upstream_status{namespace='$namespace', pod=~'apicast-$env.*', service_id='$service_id'}[1m])) by (status)"),  # noqa: E501
+    ("backend",
+     "sum(rate(apisonator_listener_response_codes{namespace=\\\"$namespace\\\",request_type=\\\"authorize\\\"}[1m]))"),  # noqa: E501
+    ("kubernetes-resources-by-namespace",
+     "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{cluster=\\\"$cluster\\\", namespace=\\\"$namespace\\\"}) by (pod)"),  # noqa: E501
+    ("kubernetes-resources-by-pod",
+     "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{namespace=\\\"$namespace\\\", pod=\\\"$pod\\\", container!=\\\"POD\\\", cluster=\\\"$cluster\\\"}) by (container)"),  # noqa: E501
+    ("system",
+     "sum(rate(rails_requests_total{namespace='$namespace',pod=~'system-app-[a-z0-9]+-[a-z0-9]+'}[1m])) by (status)"),
+    ("zync",
+     "sum(rate(rails_requests_total{namespace='$namespace',pod=~'zync-[a-z0-9]+-[a-z0-9]+'}[1m])) by (status)"),
+]
+# pylint: enable=line-too-long
+
+
+@pytest.mark.smoke
```

Comment:
Smokes need to be run even without any additional configuration of the instance, is it really the case for grafana?

Reply:
this test is checking openshift object that are created by `monitoring > enabled : true` after openshift knows about grafana definitions.

I added check for the required configuration inside of `APIManager` object.